1.8/tutorial/resources.md

```diff
@@ -0,0 +1,83 @@
+---
+post_title: DC/OS 101 - Understanding Ressources
+menu_order: 3
+---
+
+# Prerequisites
+We expect you to have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, you have just deployed app2 in our cluster and have verified it is running.
+
+# Objective
```

Comment:
In this section we will learn how to monitor and understand our resource utilization, how resource limits are enforced, and how to debug resource utilization issues.